$OpenBSD: patch-ext_sqlite3_database_h,v 1.1 2012/06/29 16:40:31 jeremy Exp $

Define a dummy RB_GC_GUARD so it works on rubinius 1.2.4.

--- ext/sqlite3/database.h.orig	Tue Jun 26 11:21:00 2012
+++ ext/sqlite3/database.h	Tue Jun 26 11:21:44 2012
@@ -12,4 +12,8 @@ typedef sqlite3Ruby * sqlite3RubyPtr;
 
 void init_sqlite3_database();
 
+#ifndef RB_GC_GUARD
+#define RB_GC_GUARD(x) x
+#endif
+
 #endif
